docs/source/upgrade/upgrade-guide-from-monitoring-3.x-to-monitoring-4.y.rst

```diff
@@ -244,24 +240,21 @@ Log in to your docker container and run the following (``start`` and ``end`` sho
                 --url http://localhost:9090 \
                 /etc/prometheus/prom_rules/back_fill/3.8/rules.1.yml
 
-It will create a ``data`` directory in the directory where you run it.
-The reason to run it under the ``/prometheus/data/`` is you can be sure Prometheus has write privileges there.
+A ``data`` directory will be created in the directory where you run the previous commands. The reason to run it under the ``/prometheus/data/`` is you can be sure Prometheus has write privileges there.
 
-    .. note::
-       Depending on the time range and the number of cores, the process can take a long time. During testing it took an hour for every week of data,
-       for a cluster with a total of 100 cores. Make sure that the creation process is not inerupt. You can split the time range to smaller durations
-       (e.g. instead of an entire year, do it a weeks at a time).
+    .. note:
+       This process may take a long time depending on the time range and number of cores. For instance, for a cluster with 100 cores, the process took an hour for    every week of data during testing. Hence, please be patient and make sure that the creation process is not interrupted. Please note the time range can be split into smaller intervals (e.g., instead of an entire year, break it down into weeks).
 
 
 Copy the data files
 ^^^^^^^^^^^^^^^^^^^
-Make sure that the process is completed successfully - don't start this section before you complete the previous sections.
+You should not start this section until all the previous sections have been completed. To copy the data files from the Docker Host to the Prometheus directory in the Docker container, run the following command:
 
 Copy the data files to the Prometheus directory:
 
 .. code-block:: bash
 
-                cp data/* .
+                docker cp {options} SRC_PATH aprom:/prometheus/data
```

Comment:
remember that in this point in time, you already inside the container at the right directory.

Reply:
Understood.